js/src/components/google-account-card/authorize-google-account-card.js

```diff
@@ -42,7 +66,27 @@ export default function AuthorizeGoogleAccountCard( { disabled = false } ) {
 		}
 	};
 
-	const description = (
+	const description = isAskingScope ? (
+		<>
+			{ additionalScopeEmail }
+			<p>
+				<em>
+					{ createInterpolateElement(
+						__(
+							'<alert>Error:</alert> You did not allow WooCommerce sufficient access to your Google account. You must select all checkboxes in the Google account modal to proceed. <link>Read more</link>',
+							'google-listings-and-ads'
+						),
+						{
+							alert: (
+								<span className="gla-authorize-google-account-card__alter-text" />
```

Comment:
❔ Would "error" text be better than "alter" text? 

```suggestion
								<span className="gla-authorize-google-account-card__error-text" />
```